tests/e2e/util.go

```diff
@@ -3333,3 +3356,29 @@ func statefulSetFromManifest(fileName string, ss *appsv1.StatefulSet) (*appsv1.S
 
 	return ss, nil
 }
+
+// collectPodLogs collects logs from all the pods in the namespace
+func collectPodLogs(ctx context.Context, client clientset.Interface, namespace string) {
+	pods, err := client.CoreV1().Pods(namespace).List(ctx, metav1.ListOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	for _, pod := range pods.Items {
+		framework.Logf("Collectiong log from pod %v", pod.Name)
+		curtime := time.Now().Unix()
+		randomValue := rand.Int()
+		val := strconv.FormatInt(int64(randomValue), 10)
+		val = string(val[1:3])
+		curtimestring := strconv.FormatInt(curtime, 10)
+		filename := curtimestring + val
+
+		//Collect Pod logs
+		for _, cont := range pod.Spec.Containers {
+			cmd := []string{"logs", "--namespace=" + namespace, pod.Name, "-c", cont.Name}
+			output, err := framework.RunKubectl(namespace, cmd...)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			framework.Logf("Writing the logs into the file %v", pod.Name+cont.Name+filename)
+			err = writeToFile(pod.Name+cont.Name+filename, output)
```

Comment:
can we put it in specific path say 'logs/*' so that we can bundle this in the job archives

Reply:
Done